tests/conftest.py

```diff
@@ -133,9 +130,8 @@ def dut_cleanup():
     TestRun.executor.run(f'{iotrace.working_dir}/tests/'
                          'security/fuzzy/fuzz.sh clean')
 
-    TestRun.LOGGER.info("Removing iotrace module")
+    TestRun.LOGGER.info("Stopping tracing")
```

Comment:
You could remove this line, because `iotrace.stop_tracing()` logs the same message.